script/Arcueid/Day 7/07_00_ARC07_1B.txt

```diff
@@ -107,52 +118,63 @@ Good morning, Noel-sensei."
 [sha:8dd2e9a6d760045ace0181f8d70c6eb65f1bc1ac]{
 -- Page 49, Offset 16694.
 -- 教室に向かう途中、ノエル先生とばったり出くわした。
-On my way to the classroom, I come across Noel-sensei unexpectedly.
+On my way to the classroom, I bump into Noel-sensei.
 }
 [sha:2d4119a894cffcc330cdd6785091e651c5e7973c]{
 -- Page 49, Offset 16695.
 -- 俺同様、こんなハンパな時間に誰かいるとは思わなかったらしく、ノエル先生はちょっと素っぽかった。
-Just like me, it seems Noel-sensei didn't expect anyone to be here at such an hour and she looks a little dumb-founded.
+// 素っ seems to be postive emphasis
+Noel-sensei seems pretty shocked to see me,
+ I suppose she wasn't expecting to run into anyone here at this hour either.
 }
 [sha:ca0b91049fc5f5f9ec9ddb311757a01db15a0e84]{
 -- Page 50, Offset 16696.
 -- 「時間を間違えると遅刻するんじゃなくて、早くから登校しちゃうんだ。志貴クンはあれね、勤勉家ね。今どきの若い子なら、もっと気楽に生きていいんじゃない？」
-"Rather than getting the time mixed up and coming to school late, you came to school early instead. You must be one of those types, the hard working ones right? But while you're still young, wouldn't it be better to loosen up a little bit?"
+// 志貴クンはあれね、勤勉家ね a bit rough to TL
+"So you lost track of time, but instead of showing up late you ended up coming to school too early.
+ That's your style, right? The diligent type.
+ You're still young though, surely it wouldn't hurt to take things a little easier while you can?"
 }
 [sha:a243e000a304d397108d24960fa42ae31e566e62]{
 -- Page 50, Offset 16697.
 -- 「十分気楽にやってますよ。勉強好きでもありませんし。
-"I'm plenty laid-back. Besides, I don't even like to study.
+"Oh I'm taking it easy, believe me. It's not like I love studying or anything.
 }
 [sha:6e735b8f46b5684a39a6157a2b8dabd6459d0202]{
 -- Page 50, Offset 16698.
 -- それより、金曜日の無断欠席についてですけど……」
-More importantly, I want to talk about my unnotified absence from last Friday..."
+More importantly, about my unexcused absence last Friday..."
 }
 [sha:3f1fb98adc423dd7ef0b2bc8abe4b8eff6d2884b]{
 -- Page 51, Offset 16699.
 -- 「ああ、アレね。大丈夫、お家には連絡入れてないから。
-"Oh, you mean that. Don't worry, I didn't contact your family.
+"Oh, that. Relax, I didn't inform your family about it.
 }
 [sha:5caf296439ad270e9234a5af9e54262f54872257]{
 -- Page 51, Offset 16700.
 -- たった一日のボイコットなんてフツーだものね。志貴クンだって、その方が都合が良かったでしょう？」
-A boycott every once in a while is totally normal. Besides, it's more convenient for you this way, right Shiki-kun?"
+// Feels like it needs the 'once in a while', you can sorta say it's derived from フツー
+It's totally normal to bunk off for a day every once in a while.
```

Comment:
hrm, "bunk off" is not something I think Noel would say. I honestly say go back to boycott here.